summoner-cli/summoner.cabal

```
@@ -109,7 +109,7 @@ library
                      , shellmet ^>= 0.0.1
                      , text ^>= 1.2.3.0
                      , time ^>= 1.8
-                     , tomland ^>= 1.0
+                     , tomland >= 1.0
```

I think having upper bounds is always nice, even for our packages. We can accidentally break something, can happen with everyone, and we don't want our own packages to break...
```suggestion
                     , tomland >= 1.0 && < 1.2
```